src/Mod/Sketcher/App/SketchObject.cpp

```diff
@@ -3819,7 +3816,7 @@ int SketchObject::split(int GeoId, const Base::Vector3d& point)
         transferConstraints(GeoId, PointPos::mid, newIds.front(), PointPos::mid);
     }
 
-    delConstraints(std::move(idsOfOldConstraints));
+    delConstraints(std::move(idsOfOldConstraints), DeleteOption::NoSolve);

```

Comment:
I recall making some similar change in either split or trim, which might have been reverting this in response to another issue. A git blame may find it.